source/applications/basic/visualization/read_zdf_vis_3d.py

```diff
@@ -50,51 +49,37 @@ def _display_depthmap(xyz):
     plt.show(block=False)
 
 
-def _get_mid_point(xyz):
-    """Calculate mid point from average of the 100 centermost points.
-
-    Args:
-        xyz: X, Y and Z images (point cloud co-ordinates)
-
-    Returns:
-        mid_point: Calculated mid point
-
-    """
-    xyz_center_cube = xyz[
-        int(xyz.shape[0] / 2 - 5) : int(xyz.shape[0] / 2 + 5),
-        int(xyz.shape[1] / 2 - 5) : int(xyz.shape[1] / 2 + 5),
-        :,
-    ]
-    return (
-        np.nanmedian(xyz_center_cube[:, :, 0]),
-        np.nanmedian(xyz_center_cube[:, :, 1]),
-        np.nanmedian(xyz_center_cube[:, :, 2]),
-    )
-
-
 def _display_pointcloud(rgb, xyz):
     """Display point cloud.
 
     Display the provided point cloud `xyz`, and color it with `rgb`.
 
-    We take the centermost co-ordinate as 'lookat' point. We assume that
-    camera location is at azimuth -pi/2 and elevation -pi/2 relative to
-    the 'lookat' point.
-
     Args:
         rgb: RGB image
         xyz: X, Y and Z images (point cloud co-ordinates)
 
     Returns None
 
     """
-    mid_point = _get_mid_point(xyz)
-    point_cloud_to_view = xyz
-    point_cloud_to_view[np.isnan(xyz[:, :, 2])] = 0
-    viewer = pptk.viewer(point_cloud_to_view)
-    viewer.attributes(rgb.reshape(-1, 3) / 255.0)
-    viewer.set(lookat=mid_point)
-    viewer.set(phi=-math.pi / 2, theta=-math.pi / 2, r=mid_point[2])
+    point_cloud_to_view = np.nan_to_num(xyz)
+    point_cloud_to_view = point_cloud_to_view.reshape(-1, 3)
+    rgb = rgb.reshape(-1, 3)
+
+    pcd = o3d.geometry.PointCloud(o3d.utility.Vector3dVector(xyz))
+    pcd.colors = o3d.utility.Vector3dVector(rgb / 255)
+
+    visualizer = o3d.visualization.Visualizer()
+    visualizer.create_window(window_name="Open3D Zivid visualizer")
```

Comment:
Do we need 'Zivid' here?

Reply:
Nope, i can remove the naming here. This is maybe better so we don't confuse customers to believe we have our own viewer.

Reply:
Exactly. I agree that it is better to remove it.